HirePro/src/main/java/com/ideas2it/hirepro/customLogger/CustomLogger.java

```diff
@@ -13,7 +12,6 @@ public CustomLogger(Class<?> className) {
     }
         
     public static void info(String message) {
-        //logger.info(message);
         logger.info("apple");
```

Comment:
What exactly you're doing in the Logger.info method

Reply:
I am having issues with custom logger. its is giving null pointer exception. I ll look into it and update once resolved